docs/sphinx/quickstart.rst

```diff
@@ -41,21 +41,21 @@ It is also recommended that you check out the latest stable tag:
 The minimum dependencies needed to use |hpx| are |boost|_ and |hwloc|_. If these
 are not available through your system package manager, see
 :ref:`boost_installation` and :ref:`hwloc_installation` for instructions on how
-to build them yourself. In addition to |boost|_ and |hwloc|_, it is recommended
+to build them yourself. In addition to Boost and HWLOC, it is recommended
```

Comment:
This change needs to be reverted. `|boost|_` and `|hwloc|_` are macros that get expanded.

Reply:
I think one of the style changes @rstobaugh converged on during GSoD is to only have a link at the first occurrence of something like `Boost` in a section or page (I'm in favor of this!). In that case we can use the form `|boost|` (i.e. no underscore at the end) which does the replacement to the correct capitalization (`Boost`) but does not produce a link.

Reply:
@sithhell as msimberg mentioned, prior to GSoD, there was no consistency to when hyperlinks were used. The guideline we decided on was that external links would only be used once per page (unless directing the reader to a specific download) and internal links would always be used for relevant terms.

Reply:
@rstobaugh I may not have been very clear about it, but I think we should still keep `|boost|` and `|hwloc|` (without the underscore) so that we get consistent capitalization more or less automatically.